arch/s390/dfltcc_deflate.c

```diff
@@ -59,6 +62,9 @@ static inline int dfltcc_can_deflate_with_params(PREFIX3(streamp) strm, int leve
 int Z_INTERNAL PREFIX(dfltcc_can_deflate)(PREFIX3(streamp) strm) {
     deflate_state *state = (deflate_state *)strm->state;
 
+	if(env_dfltcc_disabled)
```

Comment:
Here and in a few other places: missing space after if.

Reply:
will fix